rcl/include/rcl/client.h

```diff
@@ -370,6 +370,25 @@ RCL_WARN_UNUSED
 rmw_client_t *
 rcl_client_get_rmw_handle(const rcl_client_t * client);
 
+/// Check that the client is valid.
+/**
+ * The bool returned is `false` if client is invalid
+ * The bool returned is `true` otherwise.
+ * This function cannot fail.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
+ *
+ * \param[in] client pointer to the rcl client
+ * \return `true` if `client` is valid, otherwise `false`
+ */
```

Comment:
My understanding is that these functions are just to clean up the code so that, if a check is added in the future, it only needs to go in one place.

But that may be completely off-base, so I'll defer to @wjwwood.